scalafix-tests/input/src/main/scala-3next/test/explicitResultTypes/ExplicitResultTypesBase.scala

```diff
@@ -0,0 +1,46 @@
+/*
+rules = ExplicitResultTypes
+ExplicitResultTypes.memberKind = [Val, Def, Var]
+ExplicitResultTypes.memberVisibility = [Public, Protected]
+ */
+package test.explicitResultTypes
+
+import scala.language.implicitConversions
+
+object ExplicitResultTypesBase {
+  def none[T] =  None.asInstanceOf[Option[T]]
+  val a = 1 + 2
+  def b() = "a" + "b"
+  var c = 1 == 1
+  protected val d = 1.0f
+  protected def e(a: Int, b: Double) = a + b
+  protected var f = (x: Int) => x + 1
+  val f0 = () => 42
+  private val g = 1
+  private def h(a: Int) = ""
+  private var i = 22
+  val k = (1, "msg")
+  def comment(x: Int) =
+    // comment
+    x + 2
+  object ExtraSpace {
+    def * = "abc".length
+    def ! = "abc".length
+    //def foo_ = "abc".length
```

Comment:
this works fine on Scala 2 but generates invalid code on Scala 3

```scala
// scala 2
def foo_ : Int = "abc".length

// scala 3.7.0
def foo_: Int = "abc".length
```

Reply:
it looks like https://github.com/scalacenter/scalafix/issues/309 hasn't been addressed in the PC

Reply:
Scalameta handled it via https://github.com/scalameta/scalameta/commit/d2acb56632f205171f10f8fec60909a9526de2ac

Reply:
Might be worth an issue I think?

Reply:
I wanted to open an issue against the PC indeed, but I can't reproduce it on scala3 main, [the heading space does get added there](https://github.com/bjaglin/dotty/commit/843b65c201539562b2766ce89006809a6b4ea276). I need to continue the investigation.

Reply:
It looks like it was fixed in the meantime. Still broken in 3.7.0-RC2. Pretty sure there was a PR about it, but I can't find it currently.

Reply:
Oh, you're right! Thanks, I was going crazy, it didn't occur to me that it could have been fixed after 3.7.0 was branched off, because the file where the fix was made didn't change. But it's actually `isOperatorName` that was improved in https://github.com/scala/scala3/pull/22854/commits/094eaee2af7ab9b1bacca51098c9ee9242a0409e.